app/views/partials/errorManager/message.scala.html

```diff
@@ -0,0 +1,5 @@
+@(implicit messages: Messages)
+<span id="contactEmail-error" class="govuk-error-message">
```

Comment:
This ID doesn't apply to us and perhaps can be changed to something more specific or removed.